onnxscript/tests/function_libs/torch_lib/extra_opinfo.py

```diff
@@ -905,4 +922,12 @@ def make_per_sample_weight(flag, idx):
         sample_inputs_func=sample_inputs_bernoulli_p_deterministic,
         supports_out=False,
     ),
+    opinfo_core.OpInfo(
+        "unfold_extra",
+        op=lambda x, *args: x.unfold(*args),
```

Comment:
torch.ops.aten?

Reply:
copy pasted from original unfold test case in common_method_invocations.